### PR TITLE
Add route for initial implementation of hr-profile page

### PR DESCRIPTION
**Pull request by casterweb**

### CODE REVIEW

File: exadel-trainee-app/angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```
Comment (shuliakleonid) [inline exadel-trainee-app/angular.json:3]: what is it?

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
-import { NgModule } from '@angular/core';
+import {    NgModule } from '@angular/core';
```
Comment (shuliakleonid) [inline exadel-trainee-app/src/app/app-routing.module.ts:1]: To fix run script
eslint \"**/*.{ts,tsx}\" --fix --color

File: exadel-trainee-app/src/app/app.module.ts [hunk 3]
```diff
@@ -2,13 +2,15 @@ import { NgModule } from '@angular/core';
...
+import {HrProfileModule} from './models/login/hr-profile/hr-profile.module'
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HrProfileComponent } from './models/login//hr-profile/hr-profile.component';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, LoginModule, AppRoutingModule, BrowserAnimationsModule],
+  declarations: [AppComponent, HrProfileComponent],
```
Comment (shuliakleonid) [inline exadel-trainee-app/src/app/app.module.ts:12]: You don`t need component and module. We  load it with lazy loading.
loadChildren: () => import('./models/login/hr-profile/hr-profile.module').then((m) => m.HrProfileModule),

File: exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (shuliakleonid) [inline exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.component.spec.ts:1]: I think we don`t need a test file.

File: exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.component.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (shuliakleonid) [inline exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.component.ts:1]: run eslint --fix

File: exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.module.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
```
Comment (shuliakleonid) [inline exadel-trainee-app/src/app/models/login/hr-profile/hr-profile.module.ts:2]: run eslint

**Review by shuliakleonid — CHANGES_REQUESTED** [review]
fix errors

File: exadel-trainee-app/angular.json (hunk 1 above)
Comment (casterweb) [reply exadel-trainee-app/angular.json:3]: goes by default

**Pull request closed without merging**